src/twisted/internet/test/reactormixins.py

```diff
@@ -324,26 +326,30 @@ def stop():
             timedOutCall.cancel()
 
 
-    def makeTestCaseClasses(cls):
+    @classmethod
+    def makeTestCaseClasses(
+        cls: Type['ReactorBuilder']
```

Comment:
Why are these all quoted?  The names are imported, so they should be in scope?

Reply:
`ReactorBuilder` is defined in the same file, so I had to use the syntax `Class name forward references`, as described here: https://mypy.readthedocs.io/en/stable/kinds_of_types.html#class-name-forward-references

otherwise mypy would complain.

Reply:
Actually mypy did not complain.  If I didn't put ReactorBuilder in quotes, pyflakes complained with:

src/twisted/internet/test/reactormixins.py:331:19 undefined name 'ReactorBuilder'
src/twisted/internet/test/reactormixins.py:332:31 undefined name 'ReactorBuilder'

Reply:
I've changed this so SynchronousTestCase is not in quotes, but I still need 'ReactorBuilder` to be in quotes for the annotation.